src/Agent.Service/Windows/AgentService.cs

```diff
@@ -283,7 +284,14 @@ private AgentUpdateResult HandleAgentUpdate()
 
         private void WriteToEventLog(string eventText, EventLogEntryType entryType)
         {
-            EventLog.WriteEntry(EventSourceName, eventText, entryType, 100);
+            try
```

Comment:
I'm OK with this if you feel like it is appropriate. My gut is we should just let it crash, otherwise I feel like we may get into weird situations where we expect information in the event log but cant find it, and makes troubleshooting more difficult.

Should rename function to: TryWriteToEventLog